code/matt/html_css_flask/lab05/static/styles.css

```diff
@@ -0,0 +1,59 @@
+@import url("https://fonts.googleapis.com/css2?family=Amatic+SC:wght@400;700&family=Fredericka+the+Great&family=Raleway:wght@200;400;800&family=BIZ+UDPMincho&family=Raleway:wght@400;500&display=swap");
+
+@import url("https://fonts.googleapis.com/css2?family=Bangers&family=Square+Peg&display=swap");
+
+header {
+  background-color: #b12500;
+
+  font-family: "Bangers", cursive;
+  font-size: 70px;
+  font-weight: 400;
+}
+
+.container {
+  max-width: 100%
+}
+
+body {
+  font-family: Raleway, sans-serif;
+}
+
+.user-input, .form-label, legend {
+  font-weight: 600;
+}
+
+
+
+@media screen and (max-width: 1000px) {
```

Comment:
Nice responsiveness. In the future, the use of bootstrap or another framework could increase your development speed.

Reply:
The page is responsive, but it doesn't look as good on larger screens or even giant screens.
Bootstrap has [a nice page explaining its breakpoints](https://getbootstrap.com/docs/5.0/layout/breakpoints/#available-breakpoints) and they encompass fairly standard screen sizes. Short of using bootstrap or another framework you would want to research your end-users and determine what to support based on that.